corehq/ex-submodules/casexml/apps/phone/tasks.py

```diff
@@ -102,8 +102,10 @@ def prune_synclogs():
             year=year,
             week="%02d" % week
         )
+        truncate_query = "TRUNCATE TABLE {}".format(table_name)
         drop_query = "DROP TABLE IF EXISTS {}".format(table_name)
         db = router.db_for_write(SyncLogSQL)
         with connections[db].cursor() as cursor:
+            cursor.execute(truncate_query)
```

Comment:
if we don't truncate the space is not released on drop?

Reply:
Does it need "IF exists" check too?

Reply:
> if we don't truncate the space is not released on drop?
Right, you may read the documentation https://www.postgresql.org/docs/9.1/sql-truncate.html. 

Also it doesn't support the `IF exists` check, given that there is a row in the table, the check should not be necessary anyway.